src/MAPcomputation.m

```diff
@@ -141,17 +139,11 @@
 
     SigmaBarD_inv = D' * SigmaD_inv * D + Sigmad_inv;
     muBarD        = SigmaBarD_inv \ (Sigmad_inv * mud - D' * SigmaD_inv * b_D);
-
-%     Sigma_dgiveny{i} = inv(SigmaBarD_inv + Y' * Sigmay_inv * Y);
-    mu_dgiveny(:,i)      = (SigmaBarD_inv + Y' * Sigmay_inv * Y) \ (Y' * Sigmay_inv * (y(:,i) - b_Y) ...
+   
+    Sigma_dgiveny{i}   = inv(SigmaBarD_inv + Y' * Sigmay_inv * Y);
```

Comment:
Since here it is required the computation of 
~~~matlab 
Sigma_dgiveny{i}
~~~
the code takes too much time for the analysis of each trial.  It can be useful enabling this function with the optional possibility of saving (or not) them.